service-worker.js

```diff
@@ -0,0 +1,22 @@
+self.addEventListener("install", (event) => {
+    event.waitUntil(
+        caches.open("v1").then((cache) => {
+            return cache.addAll([
+                "./",
+                "index.html",
+                "service-worker.js",
+                "not911.webmanifest",
+                "seattle.html",
+                "404.html",
+                "style.css",
+                "phone.svg",
+                "apple-touch-icon.png",
+                "favicon.ico",
```

Comment:
As we add cities we'll need to add them here

Reply:
Additionally, we can probably actually just install _per city_ that might be better than trying to install the whole app. I think it would be possible to do this with templating but I'm not sure if we can actually template the `service-worker.js`...